### PR TITLE
Added Administator Mode page inside Advanced Section

### PR DESCRIPTION
**Pull request by rahulchheda**
Signed-off-by: Rahul M Chheda <rahul.chheda@mayadata.io>

### CODE REVIEW

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+id: version-1.2.0-admin-mode
+title: Administrator Mode
+sidebar_label: Administrator Mode
+original_id: admin-mode
+---
+------
+
+###  What is Adminstator Mode? 
+
+Admin Mode is a priviledged mode that LitmusChaos support in order to make SRE's/ Cluster Admins/ Developers life easier.
```
Comment (ksatchit) [inline website/versioned_docs/version-1.2.0/admin-mode.md]:
```suggestion
Admin mode is one of the ways the chaos orchestration is set up in Litmus, wherein all chaos resources (i.e., install time resources like the operator, chaosexperiment CRs, chaosServiceAccount/rbac and runtime resources like chaosengine, chaos-runner, experiment jobs & chaosresults) are setup in a single admin namespace (typically, litmus). In other words, centralized administration of chaos. 

This feature is aimed at making the SRE/Cluster Admins life easier by doing away with setting up chaos pre-requisites on a per namespace basis (which may be more relevant in an autonomous/self-service cluster sharing model in dev environments). 
```

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+This mode helps to induce chaos in different namespaces, by managing all the chaos resources in a single namespace.
```
Comment (ksatchit) [inline website/versioned_docs/version-1.2.0/admin-mode.md]:
```suggestion
This mode typically needs a "wider" & "stronger" ClusterRole, albeit one that is still just a superset of the individual experiment permissions. In this mode, the applications in their respective namespaces are subjected to chaos while the chaos job runs elsewhere, i.e., admin namespace. 
```

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+---
+# Source: openebs/templates/clusterrole.yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: litmus
+  labels:
+    name: litmus
+rules:
+- apiGroups: ["*"]
```
Comment (ksatchit) [inline website/versioned_docs/version-1.2.0/admin-mode.md]: Can we recommend superset permissions of all experiments -- `*/*/*` contains just too much beyond what we need.

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+------
+
+###  What is Adminstator Mode? 
+
+Admin Mode is a priviledged mode that LitmusChaos support in order to make SRE's/ Cluster Admins/ Developers life easier.
+This mode helps to induce chaos in different namespaces, by managing all the chaos resources in a single namespace.
+
+### How to use Adminstator Mode?
+
+In order to use Admin Mode, you just have to create a ServiceAccount in Chaos Resources Namespace, that has the permissions to perform operations on different kubernetes resources, in different namespaces.
```
Comment (ksatchit) [inline website/versioned_docs/version-1.2.0/admin-mode.md]:
```suggestion
In order to use Admin Mode, you just have to create a ServiceAccount in the *admin* or so called *chaos* namespace (`litmus` itself can be used), which is tied to a ClusterRole that has the permissions to perform operations on Kubernetes resources involved in the selected experiments across namespaces.
```

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+Just use this ServiceAccount Name in ChaosEngine chaosServiceAccount, this will let you to have all chaos resources in a single namespace, with chaos being performed on various namespaces.
```
Comment (ksatchit) [inline website/versioned_docs/version-1.2.0/admin-mode.md]:
```suggestion
Use provide this ServiceAccount in ChaosEngine's `.spec.chaosServiceAccount`. 
```

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+- Create the ChaosEngine manifest prepared in the previous step to trigger the Chaos.
+
+  `kubectl apply -f chaosengine.yml`
+
+### Watch Chaos Engine 
+
+- Describe Chaos Engine for chaos steps.
+
+  `kubectl describe chaosengine <chaos-engine> -n <chaos-resources-namespace>`
```
Comment (ksatchit) [inline website/versioned_docs/version-1.2.0/admin-mode.md]:
```suggestion
  `kubectl describe chaosengine nginx-chaos -n litmus`
```

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+### Watch Chaos progress
+
+- View pod terminations & recovery by setting up a watch on the pods in the application namespace
+
+  `watch -n 1 kubectl get pods -n <application-namespace>`
```
Comment (ksatchit) [inline website/versioned_docs/version-1.2.0/admin-mode.md]:
```suggestion
  `watch -n 1 kubectl get pods -n default`
```

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+### Check Chaos Experiment Result
+
+- Check whether the application is resilient to the pod failure, once the experiment (job) is completed. The ChaosResult resource name is derived like this: `<ChaosEngine-Name>-<ChaosExperiment-Name>`.
+
+  `kubectl describe chaosresult nginx-chaos-pod-delete -n <application-namespace>`
```
Comment (ksatchit) [inline website/versioned_docs/version-1.2.0/admin-mode.md]:
```suggestion
  `kubectl describe chaosresult nginx-chaos-pod-delete -n litmus`
```

**Review by ksatchit — DISMISSED** [review]

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+This feature is aimed at making the SRE/Cluster Admins life easier by doing away with setting up chaos pre-requisites on a per namespace basis (which may be more relevant in an autonomous/self-service cluster sharing model in dev environments).
+This mode typically needs a "wider" & "stronger" ClusterRole, albeit one that is still just a superset of the individual experiment permissions. In this mode, the applications in their respective namespaces are subjected to chaos while the chaos job runs elsewhere, i.e., admin namespace. 
+
+### How to use Adminstator Mode?
+
+In order to use Admin Mode, you just have to create a ServiceAccount in the *admin* or so called *chaos* namespace (`litmus` itself can be used), which is tied to a ClusterRole that has the permissions to perform operations on Kubernetes resources involved in the selected experiments across namespaces.
+Use provide this ServiceAccount in ChaosEngine's `.spec.chaosServiceAccount`.
```
Comment (ispeakc0de) [inline website/versioned_docs/version-1.2.0/admin-mode.md]: Provide this ServiceAccount in ChaosEngine's `.spec.chaosServiceAccount`.

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+### How to use Adminstator Mode?
+
+In order to use Admin Mode, you just have to create a ServiceAccount in the *admin* or so called *chaos* namespace (`litmus` itself can be used), which is tied to a ClusterRole that has the permissions to perform operations on Kubernetes resources involved in the selected experiments across namespaces.
+Use provide this ServiceAccount in ChaosEngine's `.spec.chaosServiceAccount`. 
+
+### Example
+
+#### Prepare RBAC Manifest 
+
+https://litmuschaos.github.io/pages/litmus-admin-rbac.yaml
```
Comment (ispeakc0de) [inline website/versioned_docs/version-1.2.0/admin-mode.md]: Can we make this section more descriptive or use embedmd?

File: website/sidebars.json [hunk 11]
```diff
@@ -67,6 +67,9 @@
     "Litmus FAQs": [
       "faq-general",
       "faq-troubleshooting"
+    ],
+    "Advanced": [
+      "admin-mode"
```
Comment (ispeakc0de) [inline website/sidebars.json:72]: Where is this file? Can we create a copy of this file under the docs section also?

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

**Comment by ksatchit** [comment]
Lets get one copy of the same in doc master as well.

File: website/versioned_docs/version-1.2.0/admin-mode.md (hunk 10 above)
Comment (rahulchheda) [reply website/versioned_docs/version-1.2.0/admin-mode.md]: Can you help me with embedmd? I dont have knowledge around it

File: website/sidebars.json (hunk 11 above)
Comment (rahulchheda) [reply website/sidebars.json:72]: No need, this seems to be the main directory structure of docs.

File: docs/admin-mode.md [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+### How to use Adminstator Mode?
+
+In order to use Admin Mode, you just have to create a ServiceAccount in the *admin* or so called *chaos* namespace (`litmus` itself can be used), which is tied to a ClusterRole that has the permissions to perform operations on Kubernetes resources involved in the selected experiments across namespaces.
+Provide this ServiceAccount in ChaosEngine's .spec.chaosServiceAccount. 
+
+### Example
+
+#### Prepare RBAC Manifest 
+
+https://litmuschaos.github.io/pages/litmus-admin-rbac.yaml
```
Comment (ispeakc0de) [inline docs/admin-mode.md]: Can you add also embedmd command here ??

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

File: docs/admin-mode.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+
+In order to use Admin Mode, you just have to create a ServiceAccount in the *admin* or so called *chaos* namespace (`litmus` itself can be used), which is tied to a ClusterRole that has the permissions to perform operations on Kubernetes resources involved in the selected experiments across namespaces.
+Provide this ServiceAccount in ChaosEngine's .spec.chaosServiceAccount. 
+
+### Example
+
+#### Prepare RBAC Manifest 
+
+Here is an RBAC definition, which in essence is a superset of individual experiments RBAC that has the permissions to run all chaos experiments across different namespaces.
+[embedmd]:# (https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/generic/container-kill/rbac.yaml yaml)
```
Comment (ispeakc0de) [inline docs/admin-mode.md]: This is please use the correct link, https://raw.githubusercontent.com/litmuschaos/pages/master/docs/litmus-admin-rbac.yaml

File: website/versioned_docs/version-1.2.0/admin-mode.md [hunk 14]
```diff
@@ -0,0 +1,133 @@
...
+
+In order to use Admin Mode, you just have to create a ServiceAccount in the *admin* or so called *chaos* namespace (`litmus` itself can be used), which is tied to a ClusterRole that has the permissions to perform operations on Kubernetes resources involved in the selected experiments across namespaces.
+Provide this ServiceAccount in ChaosEngine's .spec.chaosServiceAccount.
+
+### Example
+
+#### Prepare RBAC Manifest 
+
+Here is an RBAC definition, which in essence is a superset of individual experiments RBAC that has the permissions to run all chaos experiments across different namespaces.
+[embedmd]:# (https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/generic/container-kill/rbac.yaml yaml)
```
Comment (ispeakc0de) [inline website/versioned_docs/version-1.2.0/admin-mode.md]: Can you please use a correct link: https://raw.githubusercontent.com/litmuschaos/pages/master/docs/litmus-admin-rbac.yaml

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

**Review by ispeakc0de — DISMISSED** [review]
/lgtm

**Review by ksatchit — APPROVED** [review]

**Pull request merged**